app.py

```diff
@@ -0,0 +1,41 @@
+import connexion
+from config.env import env
+from connexion.resolver import MethodViewResolver
+from flask import Flask
+from openapi.utils import get_bundled_specs
+
+
+def create_app(testing=False) -> Flask:
+    connexion_app = connexion.FlaskApp(__name__, specification_dir="openapi/")
+
+    flask_app = connexion_app.app
+
+    if testing:
+        flask_app.config.from_object(
+            "config.environments.development.DevelopmentConfig"
+        )
+    else:
+        flask_app.config.from_object("config.Config")
+
+    options = {
+        "swagger_url": "/",
+    }
+    connexion_app.add_api(
+        get_bundled_specs("/openapi/api.yml"),
+        options=options,
+        resolver=MethodViewResolver("api"),
+    )
+
+    # This is needed to access the running app's environment config
+    # outside the request context using env.config.get("VARIABLE_NAME")
+    env.init_app(flask_app)
+
+    from db import db, migrate
+
+    db.init_app(flask_app)
+    migrate.init_app(flask_app, db, directory="db/migrations")
```

Comment:
I think we may need to separate this out as a separate script to run on PaaS? Does this mean the migrations will run every time the app starts?

Reply:
It looks like this only initialises command line options to the flask app - see [here](https://flask-migrate.readthedocs.io/en/latest/). Might be worth adding this as a comment or to the readme.

Reply:
Comments add to clarify